rizinorg/cutter
# Use CutterJson for Signatures and rewrite JsonModel

**Your checklist for this pull request**


**Detailed description**

This fixes a double-free in getSignatureInfo() when the string was
already freed by the intermediate CutterJson. But actually just using
this CutterJson makes more sense than the string anyway, so let's do
that.
The old JsonModel rebuilt the entire structure, defeating the main
purpose of a custom model in comparison to the existing QTreeWidget.
And since we are holding the json structures ourselves anyway, such a
custom model does not have many benefits anyway.

**Test plan (required)**

* Open `test/bins/pe/kaputty.exe`, click Certificates in the Dashboard and see the json laid out in the tree widget
* Open some other bin without signature info and see the Certificates Button in the Dashboard being disabled
